go/llvm-apple/Dockerfile.tmpl

```diff
@@ -61,11 +59,11 @@ RUN mkdir -p /tmp/osxcross
 # https://raw.githubusercontent.com/crazy-max/docker-osxcross/main/patches/lcxx.patch
 COPY lcxx.patch /tmp/osxcross/lcxx.patch
 
+COPY sdks/MacOSX10.11.sdk.tar.xz tarballs/${OSX_SDK}.tar.xz
```

Comment:
Do you think `10.11` could be replaced with a similar arg `${SDK_VERSION}`? However, it's not defined in the ARG section above, but it's already defined in `go/darwin/Dockerfile.tmpl`

Reply:
Oh good catch! It looks like through my copy/paste that I changed which SDK version was getting used here. Will fix.